packages/material-ui/src/Tooltip/Tooltip.d.ts

```diff
@@ -96,7 +96,7 @@ export interface TooltipProps
   /**
    * Tooltip title. Zero-length titles string are never displayed.
    */
-  title: React.ReactNode;
+  title: Exclude<React.ReactNode, null | undefined>;
```

Comment:
Should we expect `typescript-to-proptypes` to output `PropTypes.node.isRequired` on it's own in this configuration? cc @merceyz

Reply:
No, this is too much of a special case

Reply:
@merceyz thanks for looking at it. Ok, let's wait to see a trend in the codebase, it could likely be a one-off thing.

Reply:
> Ok, let's wait to see a trend in the codebase, it could likely be a one-off thing.

:+1: